python/src/aiconfig/editor/client/src/components/prompt/PromptsContainer.tsx

```diff
@@ -66,8 +68,10 @@ export default memo(function PromptsContainer(props: Props) {
             <Flex mt="md">
               {!readOnly && (
                 <PromptMenuButton
+                  prompt={prompt}
                   promptId={prompt._ui.id}
```

Comment:
nit: we probably don't need to pass in `promptId` anymore if passing in prompt

Reply:
Made a comment above -- we shouldn't pass prompt and we can actually just pass callbacks (and a new `showDeleteOutput` or `canDeleteOutput` prop)